tq/ssh.go

```diff
@@ -185,7 +187,7 @@ func (a *SSHAdapter) download(t *Transfer, cb ProgressCallback) error {
 		os.Remove(tmpName)
 	}()
 
-	err = a.downloadOne(t, cb, f)
+	err = a.downloadOne(t, conn, cb, f)
```

Comment:
Does the `defer` function cover the error case as well?  Maybe this could just be `return a.downloadOne(t, conn, cb, f)`?

Reply:
Yes, I think that's possible.